packages/sdk-codegen/src/typescript.gen.ts

```diff
@@ -83,7 +83,12 @@ import { IDictionary, ${this.typeNames().join(', ')} } from './models'
 export class ${this.packageName} extends APIMethods {
```

Comment:
note to self: tweak ci workflow so that python doesn't trigger on changes to language-not-python.gen.ts files